### ISSUE TITLE
site uses the old/deprecated bzapi instead of bugzilla's native REST

### ISSUE THREAD

**Issue by globau** [captainbrosset/devtools-bug-finder#21]
this site uses the 'bzapi' api to talk with bugzilla.
this api is old and deprecated - see https://wiki.mozilla.org/Bugzilla:BzAPI
bugzilla's native REST api is faster - http://bmo.readthedocs.io/en/latest/api/index.html
you should consider switching to it 😀

**Comment by captainbrosset** [comment]
Definitely, thanks for filing.
This also explains why I was a bit lost when searching for docs about the API.

**Comment by nchevobbe** [comment]
bzapi will be removed at the end of June (https://groups.google.com/forum/#!topic/mozilla.dev.platform/gDuH7RStS3c), so we should switch to the new API in the next months.
I may try to do it

**Comment by captainbrosset** [comment]
Thank you Nicolas! That would be great!
It may also be a good time to compare the feature set of this bug finder
app with the one from https://codetribute.mozilla.org/projects/devtools
If they essentially do the same thing, we could retire this app in favor of
codetribute.

The only one I can think of from the top of my head is that our bug finder
app knows how to group logical components together (like, the inspector
appears as one tool here even if it made up of many bz components).

**Comment by nchevobbe** [comment]
Good call, if there are other tools which are more active than this one, we should try to use them instead.
I filed https://github.com/mozilla-frontend-infra/codetribute/issues/383 to see what could be done for the multiple inspector modules.

**Comment by nchevobbe** [comment]
bzapi is still up, but we shouldn't rely on it.
codetribute now allows to group multiple components, and I have a PR to do that for DevTools modules: https://github.com/mozilla-frontend-infra/codetribute/pull/410
When it merges, I think we can retire this app :)

**Comment by captainbrosset** [comment]
It would be good to keep a simple HTML page running here, which essentially gives people the link to codetribute.
